source/01-EQ/05.ptx

```diff
@@ -11,8 +11,595 @@
     </objectives>
     <subsection>
     <title>Activities</title>
+
+    <definition xml:id="def-quadratic-equation">
+      <statement>
+        <p>
+          A <term>quadratic equation</term> is of the form: <me>ax^2+bx+c=0</me> 
```

Comment:
@tdegeorge Instead of a new paragraph and a `<m>` tag, just use the `<me>` tag to set an equation off on its own line.  I've gone ahead and changed it the couple places I saw this here.